projects/transformers/run.py

```diff
@@ -583,6 +633,135 @@ def run_finetuning_single_task(
     return eval_results
 
 
+def run_finetuning_squad(
+    model_args, data_args, training_args, last_checkpoint=None, run_idx=None,
+):
+    """On a single task train, evaluate, and save results"""
+
+    # Make sure dataset name, task name, and version_2_with_negative
+    # match before loading the dataset
+    data_args = check_squad_version(data_args)
+
+    data_init = init_dataset_for_squad(model_args,
```

Comment:
Another way of doing this:
```
(tokenizer, data_collator, train_dataset, eval_dataset, ...) = init_dataset_for_squad(model_args,
```

Anything you keep in parenthesis you can break down into more than one line. It is ultimately a style choice, up to you. Personally I think the one above looks cleaner and more in line with the rest of this particular codebase (the latter being a more important factor).

Reply:
I originally had it the way you described, but then I couldn't get flake8 to accept the indentation so finally I just did the brute force approach.